framework/wfa/ucc-dummy.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import socket
+import argparse
+import sys
+import threading
+import binascii
+
+def handle(host, port, respDict):
+    id = threading.get_ident()
+    print("{}: Listening on {}:{}".format(id, host, port))
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.bind((host, port))
+        s.listen()
+        conn, addr = s.accept()
+        with conn:
+            print("{}: {}@{} Connected by {}".format(id, host, port, addr))
+            while True:
+                try:
+                    data = conn.recv(1024)
+                except Exception as e:
+                    print("Exception {}".format(e))
+                    break
+                print("{}@{} Received: {}".format(host, port, data))
+                resp = [b'Status,RUNNING', b'Status,COMPLETE'] # default response
+                for key in respDict:
+                    if key in str(data):
+                        resp = respDict[key] # specific resonse
+                for r in resp:
+                    print("{}@{} Sending: {}".format(host, port, r))
+                    conn.sendall(r)
+    print("{}: {}@{} Exiting!".format(id, host, port))
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--ports", nargs='+', default=[7000, 7001, 7002, 7003, 7004, 7005, 7006, 7007, 7008, 7009, 7010, 7011, 7012, 8888])

```

Comment:
If you add `type=int` to the parameters of add_argument, you don't need the conversion below.